satpy/readers/mwr_l1b.py

```diff
@@ -0,0 +1,199 @@
+# Copyright (c) 2023, 2024 Pytroll Developers
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Reader for the level-1b data from the MWR sounder onboard AWS and EPS-STerna.
+
+AWS = Arctic Weather Satellite. MWR = Microwave Radiometer.
+
+AWS test data provided by ESA August 23, 2023.
+
+Sample data for five orbits in September 2024 provided by ESA to the Science
+Advisory Group for MWS and AWS, November 26, 2024.
+
+Sample EPS-Sterna l1b format AWS data from 16 orbits the 9th of November 2024.
+
+"""
+
+import xarray as xr
+
+from .netcdf_utils import NetCDF4FileHandler
+
+MWR_CHANNEL_NAMES = [str(i) for i in range(1, 20)]
+
+NAVIGATION_DATASET_NAMES = ["satellite_zenith_horn1",
+                            "satellite_zenith_horn2",
+                            "satellite_zenith_horn3",
+                            "satellite_zenith_horn4",
+                            "solar_azimuth_horn1",
+                            "solar_azimuth_horn2",
+                            "solar_azimuth_horn3",
+                            "solar_azimuth_horn4",
+                            "solar_zenith_horn1",
+                            "solar_zenith_horn2",
+                            "solar_zenith_horn3",
+                            "solar_zenith_horn4",
+                            "satellite_azimuth_horn1",
+                            "satellite_azimuth_horn2",
+                            "satellite_azimuth_horn3",
+                            "satellite_azimuth_horn4"]
+
+class AWS_EPS_Sterna_BaseFileHandler(NetCDF4FileHandler):
+    """Base class implementing the AWS/EPS-Sterna MWR Level-1b&c Filehandlers."""
+
+    def __init__(self, filename, filename_info, filetype_info, auto_maskandscale=True):
+        """Initialize the handler."""
+        super().__init__(filename, filename_info, filetype_info,
+                         cache_var_size=10000,
+                         cache_handle=True)
+        self.filename_info = filename_info
+
+    @property
+    def start_time(self):
+        """Get the start time."""
+        return self.filename_info["start_time"]
+
+    @property
+    def end_time(self):
+        """Get the end time."""
+        return self.filename_info["end_time"]
+
+    @property
+    def sensor(self):
+        """Get the sensor name."""
+        return self["/attr/instrument"]
```

Comment:
The sensor name in the test data is set as `"AWS"`, so it is wrong in both content and case (for Satpy, that is)
```suggestion
        return "mwr"
```

Reply:
Fixed